### PR TITLE
History rework

### PR DESCRIPTION
**Pull request by DoogeTube**
I think she is ready
- history wheel now uses flex order instead of static image tags, reducing loads
- history wheel can dynamically change size up or down without issue
- new images are placed into a freshly created img tag so there's no incorrect thumbnails
- images that exceed the history wheel buffer size are deleted forever (a really long time!)
- loading images from the history wheel works without error
- made the entire table data boxes clickable

### CODE REVIEW

File: baseSite/script.js [hunk 1]
```diff
@@ -241,29 +271,67 @@ function setupScaling() {
 //#endregion
 
 //#region manage history
-const historyBuffer = []
 
 function pushHistory(contentInfo) {
-    historyBuffer.unshift(serializeContentInfo(contentInfo))
-    if (historyBuffer.length > 30) {
-        historyBuffer.pop()
+    //slide all the images and wrap the last image to 0
```
Comment (Vertoak) [inline baseSite/script.js]: comment inaccurate

File: generator.py [hunk 2]
```diff
@@ -57,7 +57,7 @@
         getSettingsFunc = "function getSettings() {\n\tlet jsonObj = {}\n"
 
         for item in config["toggles"]:
-            toggleTable += "<td> <input type=\"checkbox\" id=\"{0}Toggle\" title=\"{1}\"> <label for=\"{0}Toggle\" title=\"{1}\">{2}</label> </td>\n".format(item["id"], item["desc"], item["name"])
+            toggleTable += "<td onclick=\"{0}Toggle.click()\"> <input type=\"checkbox\" id=\"{0}Toggle\" title=\"{1}\"> <label for=\"{0}Toggle\" title=\"{1}\">{2}</label> </td>\n".format(item["id"], item["desc"], item["name"])
```
Comment (Vertoak) [inline generator.py:60]: I don't think this onclick is ever used? The "getSettings()" function just polls the checkboxes directly, theres no tracking of the state in the background

File: baseSite/script.js [hunk 3]
```diff
@@ -92,8 +117,11 @@ function setup() {
         slider.scrollLeft = scrollLeft - walk;
         console.log(walk);
     });
+    setHistoryWheelToMaxSize()
 }
 
+
+
```
Comment (Vertoak) [inline baseSite/script.js]: uselss whitespace

File: baseSite/script.js [hunk 4]
```diff
@@ -216,6 +245,7 @@ function pushContent(contentInfo) {
     }
 
     pushHistory(currentInfo)
+
```
Comment (Vertoak) [inline baseSite/script.js]: useless whitespace

File: generator.py (hunk 2 above)
Comment (DoogeTube) [reply generator.py:60]: the onclick just checks the checkbox so you don't have to be sniper accurate to toggle an option

File: baseSite/script.js (hunk 3 above)
Comment (DoogeTube) [reply baseSite/script.js]: deleted

**Review by Vertoak — APPROVED** [review]

**Pull request merged**